web/plugin-manager/src/components/LabelButton.tsx

```diff
@@ -20,6 +22,10 @@ const LabelButton = (props: Props) => {
     function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
         event.preventDefault();
         props.callbackToParent(value);
+        setIsPending(true);
+        setTimeout(() => {
+            setIsPending(false);
+        }, 4000);
```

Comment:
so the isPending state will just stay 4 seconds an disappear ?? :/ is it really what we want

Reply:
it was a dirty temporary choice but i can spend more times on this. We have to do prop drilling on 4levels for this

(it was out of the scope at the first time)